package.json

```diff
@@ -160,5 +160,6 @@
         ]
       }
     }
-  }
+  },
```

Comment:
You'll want to remove these lines, otherwise our CI policy check tool will likely fail.